plotRisk.py

```diff
@@ -21,11 +28,10 @@ class Character:
         def __init__(self, name):
                 self.name = name
 
-        def plotRisk(self, attacks): # goes through enemies potential attacks and damage according to our defense profile Me
+        def PlotRisk(self, attacks): # goes through enemies potential attacks and damage according to our defense profile Me
                 #est time 1hr
                 #total time 30mins
                 attacksCopy = copy.deepcopy(attacks)
-                risks = ["is a high risk attack", "is a medium risk attack", "is a low risk attack", "is a no risk attack"]
 
                 global health
```

Comment:
just... why?

Reply:
Not really a reason but I figured it would make it more repeatable if needed